### DISCUSSION TITLE
科比·布莱恩特

### DISCUSSION THREAD

**Discussion by M1nam1Yume** [share121/inter-knot#2558] [其他]
![8e5faf73714d1cff75171b3b64bfe4d](https://github.com/user-attachments/assets/bea30b5b-03f3-4739-ab14-9be94971c34e)（Kobe Bryant，1978年8月23日—2020年1月26日），全名科比·比恩·布莱恩特·考克斯（Kobe Bean Bryant Cox），出生于美国宾夕法尼亚州费城，美国已故篮球运动员，司职得分后卫/小前锋。 
1996年NBA选秀，科比于第1轮第13顺位被夏洛特黄蜂队选中并被交易至洛杉矶湖人队，整个NBA生涯全部效力于洛杉矶湖人队；共夺得5次NBA总冠军、1次NBA常规赛MVP、2次NBA总决赛MVP、4次NBA全明星赛MVP、2次NBA赛季得分王；共入选NBA全明星首发阵容18次、NBA最佳阵容15次（其中一阵11次、二阵2次、三阵2次）、NBA最佳防守阵容12次（其中一阵9次、二阵3次）。 
2007年，科比首次入选美国国家男子篮球队，后帮助美国队夺得2007年美洲男篮锦标赛金牌、2008年北京奥运会男篮金牌以及2012年伦敦奥运会男篮金牌。 
2015年11月30日，科比宣布将在赛季结束后退役。 2017年12月19日，湖人队为科比举行了8号和24号双球衣退役仪式。 2018年3月，科比凭借短片《亲爱的篮球》荣获奥斯卡最佳动画短片奖。 
当地时间2020年1月26日（北京时间1月27日），科比因直升机事故遇难，时年41岁。 同年4月5日，科比入选奈史密斯篮球名人纪念堂。

**Comment by liusugong** [comment]
谁问你了？

**Comment by TThdyd** [comment]
牢大，我想你了🤧🤧

**Comment by 0662x** [comment]
爱了

**Comment by Dantehhh10** [comment]
man!

**Comment by lllliufeng** [comment]
男人！什么罐头我说！曼巴出去！

**Comment by MC-BA-Bee** [comment]
man!

**Comment by hyperion-netizen** [comment]
Man!

**Comment by 77477a** [comment]
这个委托我接了
